monai/transforms/spatial/array.py

```diff
@@ -2811,7 +3177,7 @@ def __init__(
         overlap: Union[Sequence[float], float] = 0.0,
         sort_fn: Optional[str] = None,
         threshold: Optional[float] = None,
-        pad_mode: Union[NumpyPadMode, PytorchPadMode, str] = NumpyPadMode.CONSTANT,
+        pad_mode: str = NumpyPadMode.CONSTANT,
```

Comment:
Please update all the default value of pad mode to "PytorchPadMode.CONSTANT".

Thanks.